src/encoded/static/components/body_map.js

```diff
@@ -394,7 +430,10 @@ class BodyMap extends React.Component {
         getSeriesData(unfilteredHref, this.context.fetch).then((response) => {
             const { facets } = response;
             const organFacets = facets.filter((f) => f.field === organField)[0].terms.map((f) => f.key);
-            const systemFacets = facets.filter((f) => f.field === systemsField)[0].terms.map((f) => f.key);
+            let systemFacets = [];
```

Comment:
The code you did is fine. But if you care about nano seconds of speed, you could do -

`const systemFacets = [];`

and in line 434, push the new values into the array as opposed to create a new array